src/components/button/button.jsx

```diff
@@ -0,0 +1,13 @@
+import "./button.scss";
+
+export const Button = ({ children, color, outline, border }) => {
+  return (
+    <button
+      className={`btn ${color !== undefined ? color : "primary"} ${
+        outline !== undefined ? outline : "none"
+      } ${border !== undefined ? border : "default"}`}
+    >
```

Comment:
Having so much inline code inside the `className` property can be hard to read. A better approach is to instead have an array of the classes, and then convert it to a string, joined by spaces, like so:

```
const classNames = ['btn',  color, outline, border].join(' ')

<button className={classNames}>{children}</button>
```